modules/video/test/ocl/test_optflowpyrlk.cpp

```diff
@@ -121,13 +124,24 @@ OCL_TEST_P(PyrLKOpticalFlow, Mat)
             float errdiff = 0.0f;
 
             if (!eq || errdiff > 1e-1)
+            {
                 ++mistmatch;
+                continue;
+            }
+
+            eq = std::abs(cpuErr[i] - err[i]) < 0.01;
+            if(!eq &&
+               winSize.width >= 16 && winSize.height >=16 &&
+               winSize.width <=24 && winSize.height <=24)
```

Comment:
So, How about that I add a new OpenCL kernel file in modules/video/src/opencl/ directory? When the host code check winSize within `16*16` to `24*24`, the new kernel will be called. Otherwise, the old kernel - pyrlk.cl - will be used. Is that OK?